src/components/LoaderError/LoaderError.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export const LoadingError: React.FC = () => (
+  <div className="notification is-danger is-light">
+    Oh no!
```

Comment:
There's a typo in the text. Replace 'It i&apos;s not your fault' with 'It's not your fault'. 🖊️